ExtractUserDataActivity/handler.ts

```diff
@@ -583,5 +580,6 @@ export function createExtractUserDataActivityHandler(
             value: archiveInfo
           })
       )
-      .run();
+      .run()
+      .then(e => e.value);
```

Comment:
you cannot return an Either from an activity since the return value must be serializable